content/tutorial/03-umbrella-sdk/01-unified-approach/01-simple-setup/README.md

```
@@ -0,0 +1,98 @@
+- **📦 Batteries Included**: All functionality in one place
+
+## Basic Initialization
+
+```typescript
+/// file: run.ts
+import type { Web3Provider } from '@ethersproject/providers';
+import { CowSdk, SupportedChainId } from '@cowprotocol/cow-sdk';
+import { EthersV5Adapter } from '@cowprotocol/sdk-ethers-v5-adapter';
+
+export async function run(provider: Web3Provider): Promise<unknown> {
+	const signer = provider.getSigner();
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)